src/services/activities/queries.ts

```diff
@@ -0,0 +1,9 @@
+import { gql } from 'graphql-request'
+
+export const GET_LATEST_ACTIVITIES = gql`
+    query GetLatestActivities {
+        wikis{
```

Comment:
```suggestion
        wikis {
```

my job is done here XD